src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,46 @@
+import {removeTestUser} from '@webex/test-users';
+
+import {createIntegrationTestUser, createIntegrationSdkInstance} from './testHelper';
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter', () => {
+  let roomsSDKAdapter;
+  let sdk;
+  let user;
+  let createdRoom;
+
+  beforeAll(async () => {
+    // Setting timeout higher for user creation and room creation
+    jest.setTimeout(15000);
+    user = await createIntegrationTestUser();
+    sdk = createIntegrationSdkInstance(user.token);
+    createdRoom = await sdk.rooms.create({title: 'Webex Teams Test Room'});
+  });
+
+  beforeEach(() => {
+    roomsSDKAdapter = new RoomsSDKAdapter(sdk);
+  });
+
+  describe('getRoom() returns', () => {
+    test('a room in a proper shape', (done) => {
+      roomsSDKAdapter.getRoom(createdRoom.id).subscribe((room) => {
```

Comment:
I think the reason why the `jest` process doesn't finish is because we're not unsubscribing.

Reply:
I'll update the test to unsubscribe, but it still isn't finishing properly:

>Jest did not exit one second after the test run has completed.

>This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.